SCons/Tool/packaging/__init__.py

```diff
@@ -86,8 +86,7 @@ def Tag(env, target, source, *more_tags, **kw_tags):
             t.Tag(k, v)
 
 def Package(env, target=None, source=None, **kw):
-    """ Entry point for the package tool.
-    """
+    """Entry point for the package tool."""
```

Comment:
Does this have anything to do with OverrideEnvironment() changes?

Reply:
Not the reformatted comment, obviously, but the two changed tools now call the `Override` factory to get their OE rather than instantiating an `OverrideEnvironment` directly, which looks to be generally preferable since the former handles some of the "magic" involved.